### PR TITLE
[1.19] Update wills

### PR DESCRIPTION
**Pull request by Aegide**
I could make the tooltips even smaller, but it's already "good enough".
<br>
# New Terrasteel Helmet
helmet_new
<br>
# Current Terrasteel Helmet
helmet_old
<br>
# New Will
will_new
<br>
# Current Will
will_old

### CODE REVIEW

**Comment by williewillus** [comment]
Target 1.19 please.

**Review by Aegide — COMMENTED** [review]
Ok should be better.

File: Xplat/src/main/resources/assets/botania/lang/en_us.json [hunk 1]
```diff
@@ -62,7 +62,7 @@
   "botaniamisc.hasClip": "\u00a76Has Lens Clip",
   "botaniamisc.clipEmpty": "(Empty)",
   "botaniamisc.armorset": "\u00a7bArmor Set\u00a7r:",
-  "botaniamisc.craftToAddWill": "Craft with a Terrasteel Helmet to add the following effect:",
+  "botaniamisc.craftToAddWill": "Terrasteel Helmet upgrade:",
```
Comment (williewillus) [inline Xplat/src/main/resources/assets/botania/lang/en_us.json]: capitalize Upgrade

Comment (Aegide) [reply Xplat/src/main/resources/assets/botania/lang/en_us.json]: Sure, done.

**Pull request merged**